examples/k8s.yaml

```diff
@@ -113,12 +126,14 @@ spec:
     port: 55678
     protocol: TCP
     targetPort: 55678
-#  - name: jaeger-tchannel
-#    port: 14267
-#  - name: jaeger_thrift_http
-#    port: 14268
-#  - name: zipkin
-#    port: 9411
+  - name: jaeger-tchannel
+    port: 14267
+  - name: jaeger_thrift_http
```

Comment:
nit: for consistency let's use either `-` or `_`

Reply:
I'll use lower case for consistency - but it doesn't matter too much for this as they are just the names for the ports.